releasenotes/index.md

```diff
@@ -6,6 +6,7 @@ permalink: /releasenotes/
 
 The Release Notes and Javadocs for every major release of Apache jclouds are listed chronologically below.
 
+* [1.9.2](/releasenotes/1.9.2) | [Javadoc](/reference/javadoc/1.9.x/)
```

Comment:
Ideally option 3, but I recall some discussions about the amount of space required to host that, so given that most javadoc will not change in minor versions (at least it shouldn't) I'd say we should go for option 2.